src/Components/users/users.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+function Users() {
+  const [results, setResults] = useState([]);
+
+
+
+    async function fetchApi () {
+      try{
+        const results = await axios({method:'get', url: 'http://my-json-server.typicode.com/alexdr00/mock-data/users'});
+        setResults (results.data)
+      }catch (error) {
+        console.log(error)
```

Comment:
Console logging with error is not enough, a client never uses the console to se what went wrong.
A warning component would be better suited to show to the user the errors.